include/engine/routing_algorithms/routing_base.hpp

```diff
@@ -630,7 +630,7 @@ template <class DataFacadeT, class Derived> class BasicRoutingInterface
         NodeID last_id = SPECIAL_NODEID;
         forward_core_heap.Clear();
         reverse_core_heap.Clear();
-        for (const auto p : forward_entry_points)
+        for (const auto& p : forward_entry_points)
```

Comment:
Will fix. I just didn't want to run clang-format on the whole file, otherwise we get a lot of noise in here..